valve-monitor/web-app/src/styles/Layout.module.scss

```diff
@@ -16,7 +16,6 @@
     ) !important;
     width: clamp($page-content-xs, $page-content-lg, calc(100vw - 79px));
     margin: 0 auto;
-    background-color: #fff;
```

Comment:
I believe the white inner background is part of the design mockups now. I'd leave this in

Reply:
I went ahead moved the `background-color: #fff` up to `.mainContentWrapper` per our conversation with Michelle.